src/learn/math/drill/badge.jsx

```diff
@@ -1,39 +1,25 @@
-const FloatingActionButton = require('material-ui/FloatingActionButton').default;
+const Fab = require('@material-ui/core/Fab').default;
```

Comment:
While I agree with the renaming of the variable (component) to `Fab` - can we keep it at `FloatingActionButton` for this PR and do it in a later one. That way there are fewer lines that are changing in this PR.

Reply:
Done!